doc/classes/Popup.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Popup" inherits="Window" version="4.0">
 	<brief_description>
-		Base container control for popups and dialogs.
+		Base container for popups and dialogs.
 	</brief_description>
 	<description>
-		Popup is a base [Control] used to show dialogs and popups. It's a subwindow and modal by default (see [Control]) and has helpers for custom popup behavior.
+		Popup is a base container used to show dialogs and popups. It's a subwindow and modal by default and has helpers for custom popup behavior.
```

Comment:
Popups aren't used for dialogs anymore, so that bit needs to be removed as well. I recommend wording it like this:

> Popup is a base window container for popup-like subwindows.

> Popup is a base window container for popup-like subwindows. It's a modal by default (see [member close_on_parent_focus]) and has helpers for custom popup behavior.

The `close_on_parent_focus` description could also be modified to be like:

> If true, the [Popup] will close when its parent [Window] is focused.

And `popup_hide`:

> Emitted when the popup is hidden.

Reply:
Thanks, included your reccomendations